src/test/integration/tokens.e2e-spec.ts

```diff
@@ -111,4 +123,73 @@ describe('Token Service', () => {
       expect(await tokenService.getToken(tokenIdentifier + 'a')).toBeUndefined();
     });
   });
-});
+
+  describe('Get Token Roles', () => {
+    it(`should return token roles`, async () => {
+      const tokenRoles = await tokenService.getToken(tokenIdentifier);
+
+      if (tokenRoles) {
+        const value = await tokenService.getTokenRoles(tokenRoles.identifier);
+        expect(value).toBeInstanceOf(Array);
+      }
+    });
+  });
+
+  describe('Get Token Accounts', () => {
+    it(`should return token with size of 10`, async () => {
```

Comment:
Please update test description and variables

Reply:
Change request resolved with commit 25855b729cdda8a4182fa5de64732490d82fc3b6